sourabh/Stuff I would like to do someday.html

```diff
@@ -0,0 +1,15 @@
+<html>
```

Comment:
you've used the 8 space of tab, reduce it by 4, i.e. make 4 space of tab, and code accordingly.
Also where is the "!DOCTYPE". Do you have any idea about "!DOCTYPE"?